tests/test_suggestions.py

```diff
@@ -18,51 +18,75 @@
 from tests.conftest import setup_pyspark
 
 
+def _create_spark_dataframe(spark_context):
+    column_a = ["foo"] * 10 + ["bar"] * 10 + ["baz"] * 10
+    column_b = [1] * 5 + [2] * 15 + [3] * 10
+    column_c = [5] * 15 + [6] * 10 + [None] * 5
+    column_d = list(range(30))
+    column_e = [True] * 10 + [False] * 20
+    rows = [
+        Row(a=a, b=b, c=c, d=d, e=e)
+        for a, b, c, d, e in zip(column_a, column_b, column_c, column_d, column_e)
+    ]
+    return spark_context.parallelize(rows).toDF()
+
+
 class TestSuggestions(unittest.TestCase):
     @classmethod
     def setUpClass(cls):
         cls.spark = setup_pyspark().appName("test-analyzers-local").getOrCreate()
         cls.ConstraintSuggestionRunner = ConstraintSuggestionRunner(cls.spark)
         cls.sc = cls.spark.sparkContext
-        cls.df = cls.sc.parallelize([Row(a="foo", b=1, c=5), Row(a="bar", b=2, c=6), Row(a="baz", b=3, c=None)]).toDF()
+        cls.df = _create_spark_dataframe(cls.sc)
```

Comment:
Ah.. we are still using vanilla unittest. Do you mind change the test runner to pytest and use [fixtures](https://docs.pytest.org/en/6.2.x/fixture.html) for those (and probably spark session as well). It should be quite handy. 

Don't have to migrate everything, it should run `unittest.TestCases` just fine. And separate PR will be great if you like

Reply:
Yep! I can do that in a future PR to make it easier to review 😃